ipynb/releases/ReleaseNotes_v16.12.ipynb

```diff
@@ -0,0 +1,778 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "# Target Connectivity"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true
+   },
+   "source": [
+    "## Configurable logging system"
```

Comment:
I not sure it's right to have this under "Target connectivity" as the "Configurable logging system" applies to all modules.